resources/function-controller/README.md

```diff
@@ -53,6 +54,7 @@ Follow the steps to install the chart:
                  --set secret.registryUserName="${reg_username}" \
                  --set secret.registryPassword="${reg_password}" \
                  --set config.dockerRegistry="${FN_REPOSITORY}" \
+                 --set global.ingress.domainName="${DOMAIN_NAME}" \
```

Comment:
Where is this references?

Reply:
@antoineco It's already on master, but we forgot to put it in the README file before. 
It's needed in the microfrontend introduced in this commit: https://github.com/kyma-project/kyma/commit/11b46c65fc0ff56ee3f8921d5e58717914df52ae